src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -1948,6 +1948,35 @@ public void setBoots(@NotNull ItemStack itemStack) {
         inventory.setBoots(itemStack);
     }
 
+    /**
+     * Sets a cooldown on a material (like when an enderpearl is used)
+     *
+     * @param material the material to add a cooldown to
+     * @param ticks how long the material will be on cooldown
+     */
+    public void setCooldown(Material material, int ticks) {
+        setCooldown(material, ticks, true);
+    }
+
+    /**
+     * Sets a cooldown on a material (like when an enderpearl is used)
+     *
+     * @param material the material to add a cooldown to
+     * @param ticks how long the material will be on cooldown
+     * @param lagCompensation whether another packet is sent later to remove cooldown (incase the server lags)
+     */
+    public void setCooldown(Material material, int ticks, boolean lagCompensation) {
+        final SetCooldownPacket packet = new SetCooldownPacket(material.id(), 0);
```

Comment:
Shouldn't the second argument be `ticks` rather than 0?

Reply:
whoops yep